src/DevBetterWeb.Web/DevBetterWeb.Web.csproj

```diff
@@ -19,6 +19,9 @@
 		<Content Update="wwwroot\**\*;Views\**\*;Areas\**\Views;appsettings.json;web.config">
 			<CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
 		</Content>
+		<Content Include="..\..\.dockerignore">
+		  <Link>.dockerignore</Link>
+		</Content>
```

Comment:
Is this section required?

Reply:
It's not. I removed it in the last commit just now